### PR TITLE
B 20991 main - Add Counseling Office Column and Filtering to TIO Queue

### PR DESCRIPTION
**Pull request by JonSpight**
## Summary
When a customer selects an origin duty location assigned to government counseling, then they will also be asked to select a counseling office within the same GBLOC. This field will be called "Counseling Office".

How to test
1. Login as a customer and create a 2 both with different counseling offices.
2. Login as a Services Counselor and submit the new move.
3 Login as TOO, approve both moves.
4. Crete multiple payment requests for each move
5. Login as TIO and filter the Counseling Office Task Order Queue for a move to verify filtering works.
6. Login as TIO and sort the Counseling Office Task Order Queue for a move to verify sort works.

## Screenshots
[B-20991-508Screenshots.pdf](https://github.com/user-attachments/files/17740707/B-20991-508Screenshots.pdf)

### CODE REVIEW

File: src/pages/Office/ServicesCounselingQueue/ServicesCounselingQueue.jsx [hunk 1]
```diff
@@ -262,7 +262,7 @@ export const closeoutColumns = (
         );
       },
       {
-        id: 'lastName',
+        id: 'customerName',
         isFilterable: true,
         exportValue: (row) => {
```
Comment (loganwc) [inline src/pages/Office/ServicesCounselingQueue/ServicesCounselingQueue.jsx:267]: missing changes from your integration PR
image

https://github.com/transcom/mymove/pull/14185/files#diff-ae0734b17c3ed4618319fa9001a0f17f44a056a0804853a02fa3386f862c64c2

**Review by loganwc — CHANGES_REQUESTED** [review]

File: src/pages/Office/ServicesCounselingQueue/ServicesCounselingQueue.jsx (hunk 1 above)
Comment (JonSpight) [reply src/pages/Office/ServicesCounselingQueue/ServicesCounselingQueue.jsx:267]: fixed

**Review by danieljordan-caci — APPROVED** [review]
Commits are a little wonky, but I'm assuming you had to do some merge conflicts when going into INT.

Code matches up 1:1 though, so LGTM.

**Review by loganwc — APPROVED** [review]
awesome

**Review by pambecker — APPROVED** [review]

**Pull request merged**